tests/playTests.sh

```diff
@@ -620,6 +620,23 @@ else
     $ECHO "lz4 mode not supported"
 fi
 
+$ECHO "\n===> suffix list test"
+
+$ZSTD -d tmp.abc 2> tmplg || echo 
```

Comment:
I suspect you use `|| echo` in order to cancel the return value of `$ZSTD` which just failed.

There is simpler and more useful : 
`! $ZSTD -d tmp.abc`
The will reverse the success / failure signal.
And `$ZSTD -d tmp.abc` must indeed fail.